2020-03-29 14:51:45,571 INFO: [SERVER] Server configuration file reset successfully
2020-03-29 15:12:20,898 INFO: [SERVER] Group 'bob' added successfully
2020-03-29 15:12:30,450 INFO: [SERVER] Group 'Bob' added successfully
2020-03-29 15:12:41,743 INFO: [SERVER] Group 'Bob' added successfully
2020-03-29 15:13:04,860 INFO: [SERVER] Group 'Bob' added successfully
2020-03-29 15:15:03,248 INFO: [SERVER] Group 'Bob' added successfully
2020-03-29 15:15:04,730 ERROR: [SERVER] Group 'Bob' already exists
2020-03-29 15:15:06,008 ERROR: [SERVER] Group 'Bob' already exists
2020-03-29 15:15:06,913 ERROR: [SERVER] Group 'Bob' already exists
2020-03-29 15:20:23,856 ERROR: [SERVER] Group 'g' does not exist
2020-03-29 15:20:27,606 INFO: [SERVER] Group 'Bob' removed successfully
2020-03-29 15:20:37,537 INFO: [SERVER] Group 'Bob' added successfully
2020-03-29 15:21:07,968 ERROR: [SERVER] Group 'Bob' already exists
2020-03-29 15:21:13,524 INFO: [SERVER] Group 'Bob' removed successfully
2020-03-29 15:21:14,331 INFO: [SERVER] Group 'Bob' added successfully
2020-03-29 15:22:39,864 INFO: [SERVER] Group 'Bob' removed successfully
2020-03-29 15:22:40,946 INFO: [SERVER] Group 'Bob' added successfully
2020-03-29 15:30:02,774 INFO: [SERVER] Group 'dev' added successfully
2020-03-29 15:30:06,768 INFO: [SERVER] Group 'dev' removed successfully
2020-03-29 15:33:23,532 INFO: [SERVER] Group 'bob' added successfully
2020-03-29 15:33:49,357 INFO: [SERVER] Group 'bob' removed successfully
2020-03-29 15:33:55,486 INFO: [SERVER] Group 'Bob' removed successfully
2020-03-29 16:36:53,887 INFO: [SERVER] User 'joe' added successfully
2020-03-29 16:37:28,412 INFO: [SERVER] User 'joe' added successfully
2020-03-29 16:37:48,608 INFO: [SERVER] Group 'dev' added successfully
2020-03-29 16:38:20,544 INFO: [SERVER] User 'joe' added successfully
2020-03-29 16:38:22,162 INFO: [SERVER] Group 'dev' added successfully
2020-03-29 16:40:49,368 INFO: [SERVER] User 'joe' removed successfully
2020-03-29 16:40:51,169 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:40:52,042 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:40:52,483 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:40:52,769 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:40:53,019 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:40:53,237 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:40:53,457 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:40:53,663 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:40:53,840 ERROR: [SERVER] User 'joe' does not exist
2020-03-29 16:41:05,336 INFO: [SERVER] Group 'dev' removed successfully
2020-03-29 17:40:49,972 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 17:40:49,972 INFO: [foo] Plugin started successfully!
2020-03-29 17:41:35,465 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 17:41:35,465 INFO: [foo] Plugin started successfully!
2020-03-29 17:41:35,465 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 17:42:18,115 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 17:42:18,115 INFO: [foo] Plugin started successfully!
2020-03-29 17:42:27,968 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 17:42:27,969 INFO: [foo] Plugin started successfully!
2020-03-29 17:42:27,969 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 17:53:04,102 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 17:53:04,102 INFO: [foo] Plugin started successfully!
2020-03-29 17:53:04,102 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 19:09:45,116 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 19:10:05,045 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 19:10:33,614 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 19:10:33,616 INFO: [foo] Plugin started successfully!
2020-03-29 19:10:33,616 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 19:10:43,328 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 19:10:43,330 INFO: [foo] Plugin started successfully!
2020-03-29 19:10:43,330 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 19:10:55,345 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 19:11:07,192 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 19:11:07,193 INFO: [foo] Plugin started successfully!
2020-03-29 19:11:07,193 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 19:11:16,926 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 19:11:43,885 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 19:11:43,887 INFO: [foo] Plugin started successfully!
2020-03-29 19:11:43,887 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 21:32:39,097 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 21:32:39,098 INFO: [foo] Plugin started successfully!
2020-03-29 21:32:39,098 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 21:38:59,276 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 21:39:04,660 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 21:39:04,662 INFO: [foo] Plugin started successfully!
2020-03-29 21:39:56,634 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 21:39:56,636 INFO: [foo] Plugin started successfully!
2020-03-29 21:40:22,256 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 21:40:22,258 INFO: [foo] Plugin started successfully!
2020-03-29 21:40:22,258 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 21:40:44,425 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 21:40:44,427 INFO: [foo] Plugin started successfully!
2020-03-29 21:40:44,427 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:24:33,763 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:24:33,764 INFO: [foo] Plugin started successfully!
2020-03-29 23:24:33,765 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:26:24,399 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:26:24,400 INFO: [foo] Plugin started successfully!
2020-03-29 23:26:24,401 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:26:51,812 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in run_server
    data = await socket.recv()
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/protocol.py", line 509, in recv
    await self.ensure_open()
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/protocol.py", line 812, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedOK: code = 1001 (going away), no reason
2020-03-29 23:29:11,137 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:29:13,465 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:29:36,780 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:29:36,782 INFO: [foo] Plugin started successfully!
2020-03-29 23:29:36,783 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:31:16,011 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 24, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 53, in parse_packet
    return js.loads(x)
NameError: name 'js' is not defined
2020-03-29 23:31:41,048 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 24, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 53, in parse_packet
    return js.loads(x)
NameError: name 'js' is not defined
2020-03-29 23:32:50,122 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:32:50,123 INFO: [foo] Plugin started successfully!
2020-03-29 23:32:50,124 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:33:04,524 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 24, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 50, in parse_packet
    return js.loads(x)
NameError: name 'js' is not defined
2020-03-29 23:33:42,554 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:33:42,555 INFO: [foo] Plugin started successfully!
2020-03-29 23:33:42,555 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:34:00,207 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 50, in parse_packet
    return js.loads(x)
NameError: name 'js' is not defined
2020-03-29 23:35:00,321 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:35:00,323 INFO: [foo] Plugin started successfully!
2020-03-29 23:35:00,323 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:35:10,875 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 50, in parse_packet
    return js.loads(x)
NameError: name 'js' is not defined
2020-03-29 23:35:59,358 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:35:59,360 INFO: [foo] Plugin started successfully!
2020-03-29 23:35:59,360 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:36:07,662 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 50, in parse_packet
    return json.loads(x)
NameError: name 'json' is not defined
2020-03-29 23:36:40,119 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:36:40,120 INFO: [foo] Plugin started successfully!
2020-03-29 23:36:40,120 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:36:52,279 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in run_server
    console.log(data)
NameError: name 'console' is not defined
2020-03-29 23:37:44,235 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:37:44,237 INFO: [foo] Plugin started successfully!
2020-03-29 23:37:44,237 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:37:53,748 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 22, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 49, in parse_packet
    return json.loads(x)
NameError: name 'json' is not defined
2020-03-29 23:39:04,581 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:39:04,582 INFO: [foo] Plugin started successfully!
2020-03-29 23:39:04,583 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:39:12,704 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 50, in parse_packet
    return json.loads(x)
NameError: name 'json' is not defined
2020-03-29 23:40:10,124 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:40:10,125 INFO: [foo] Plugin started successfully!
2020-03-29 23:40:10,125 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:40:14,128 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 50, in parse_packet
    print(json.loads(x))
NameError: name 'json' is not defined
2020-03-29 23:40:36,834 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:40:36,835 INFO: [foo] Plugin started successfully!
2020-03-29 23:40:36,835 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:40:42,177 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 41, in run_server
    packets = parse_packet(data)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 30, in parse_packet
    print(json.loads(x))
NameError: name 'json' is not defined
2020-03-29 23:41:06,838 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:41:06,839 INFO: [foo] Plugin started successfully!
2020-03-29 23:41:06,840 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:41:16,025 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 38, in run_server
    data = await socket.recv()
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/protocol.py", line 509, in recv
    await self.ensure_open()
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/protocol.py", line 812, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedOK: code = 1001 (going away), no reason
2020-03-29 23:41:26,606 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 42, in run_server
    if packet["type"] == "cmd":
TypeError: string indices must be integers
2020-03-29 23:42:51,257 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:42:51,258 INFO: [foo] Plugin started successfully!
2020-03-29 23:42:51,258 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:42:58,125 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 46, in run_server
    response = cmds.process(packet)
TypeError: process() missing 2 required positional arguments: 'socket' and 'quiet'
2020-03-29 23:46:10,007 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:46:10,008 INFO: [foo] Plugin started successfully!
2020-03-29 23:46:10,009 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:46:19,555 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 50, in run_server
    response = cmds.process(packet, socket, quiet_mode)
  File "/home/joe/Projects/WSServerBase/wssb/cmds.py", line 26, in process
    return cmd_identify(cmd, socket, quiet)
  File "/home/joe/Projects/WSServerBase/wssb/cmds.py", line 40, in cmd_identify
    logging.info("[SERVER] User '" + user_name + "' has been added to the list of connected users")
NameError: name 'user_name' is not defined
2020-03-29 23:46:56,733 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:46:56,734 INFO: [foo] Plugin started successfully!
2020-03-29 23:46:56,735 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:49:34,993 INFO: [SERVER] User 'joe' has been added to the list of connected users
2020-03-29 23:51:55,467 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:51:55,468 INFO: [foo] Plugin started successfully!
2020-03-29 23:51:55,468 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:52:05,120 INFO: [SERVER] User 'joe' has been added to the list of connected users
2020-03-29 23:52:05,120 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 56, in run_server
    await socket.send(format_packet(response))
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in format_packet
    if type(d) in (list, dict):
NameError: name 'd' is not defined
2020-03-29 23:52:57,474 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:52:57,476 INFO: [foo] Plugin started successfully!
2020-03-29 23:52:57,476 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:53:09,315 INFO: [SERVER] User 'joe' has been added to the list of connected users
2020-03-29 23:53:09,315 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 58, in run_server
    await socket.send(format_packet(response))
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 23, in format_packet
    if type(d) in (list, dict):
NameError: name 'd' is not defined
2020-03-29 23:53:37,653 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:53:37,654 INFO: [foo] Plugin started successfully!
2020-03-29 23:53:37,655 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:53:47,731 INFO: [SERVER] User 'joe' has been added to the list of connected users
2020-03-29 23:55:21,873 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:55:21,874 INFO: [foo] Plugin started successfully!
2020-03-29 23:55:21,874 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:55:30,245 INFO: [SERVER] User 'joe' has been added to the list of connected users
2020-03-29 23:55:58,758 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 46, in run_server
    data = await socket.recv()
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/protocol.py", line 509, in recv
    await self.ensure_open()
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/protocol.py", line 812, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedOK: code = 1001 (going away), no reason
2020-03-29 23:56:21,176 INFO: [SERVER] Loaded server configuration file successfully
2020-03-29 23:56:21,177 INFO: [foo] Plugin started successfully!
2020-03-29 23:56:21,177 INFO: [SERVER] Starting WebSocket server on localhost:8765
2020-03-29 23:56:36,707 INFO: [SERVER] User 'joe' has been added to the list of connected users
2020-03-29 23:56:46,941 ERROR: Error in connection handler
Traceback (most recent call last):
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/server.py", line 191, in handler
    await self.ws_handler(self, path)
  File "/home/joe/Projects/WSServerBase/wssb/core.py", line 46, in run_server
    data = await socket.recv()
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/protocol.py", line 509, in recv
    await self.ensure_open()
  File "/home/joe/miniconda3/lib/python3.7/site-packages/websockets/protocol.py", line 812, in ensure_open
    raise self.connection_closed_exc()
websockets.exceptions.ConnectionClosedOK: code = 1001 (going away), no reason
